flink-core/src/test/java/org/apache/flink/api/common/typeutils/SerializerTestInstance.java

```diff
@@ -20,14 +20,12 @@
 
 import org.apache.flink.testutils.DeeplyEqualsChecker;
 
-import org.junit.Ignore;
-import org.junit.Test;
+import org.junit.jupiter.api.Test;
 
 import java.lang.reflect.InvocationTargetException;
 import java.lang.reflect.Method;
 
-@Ignore
-public class SerializerTestInstance<T> extends SerializerTestBase<T> {
+public abstract class SerializerTestInstance<T> extends SerializerTestBase<T> {
```

Comment:
This was required because `@Disable` no longer made junit ignore the test for some reason.